app/components/explorerMetrics.tsx

```diff
@@ -1,46 +1,49 @@
 import * as React from 'react';
 
 import { useTranslation } from 'react-i18next';
-import { Icon, IconName, Card, Elevation } from '@blueprintjs/core';
+import {
+  Icon,
+  IconName,
+  Card,
+  Elevation,
+  Tooltip,
+  Position
+} from '@blueprintjs/core';
+import AnimalsListTooltipContent from './AnimalsListTooltipContent';
 
 type Props = {
   data: Observation[];
   rareTargets: string[];
-};
-
-type entry = {
-  label: string;
-  value: string;
+  emptyClasses: string[];
 };
 
 export default function ExplorerMetrics(props: Props) {
   const { t } = useTranslation();
-  const { data, rareTargets } = props;
+  const { data, rareTargets, emptyClasses } = props;
 
   const getUniqueSet = (dataset: string[]) => {
-    return Array.from(new Set(dataset)).map((entry: string) => {
-      return { value: entry, label: entry.replace(/_/g, ' ') };
-    });
+    return Array.from(new Set(dataset));
   };
 
-  const getRareAnimals = (dataset: entry[], rares: string[]) => {
-    return dataset.filter(entry => rares.includes(entry.value));
+  const getRareAnimals = (dataset: string[], rares: string[]) => {
+    return dataset.filter(entry => rares.includes(entry));
   };
 
   const nonEmpty = data.filter(
-    (entry: Observation) => !['Blank', 'empty'].includes(entry.pred_1)
+    (entry: Observation) => !emptyClasses.includes(entry.pred_1)
   );
   const uniqueAnimals = getUniqueSet(
     data.map((entry: Observation) => entry.pred_1)
   );
   const rareAnimals = getRareAnimals(uniqueAnimals, rareTargets);
 
-  const metricsCard = (
+  function metricsCard(
     icon: IconName,
     color: string,
     title: string,
-    value: number
-  ) => {
+    value: number,
+    tooltip: JSX.Element | string = ''
```

Comment:
I'm not sure, but maybe `ReactNode` is more appropriate here?

Reply:
This is exactly the type that is expected downstream